backend/roles/role-managers/active-mixplay-users.js

```diff
@@ -0,0 +1,142 @@
+"use strict";
+
+const { ipcMain } = require("electron");
+const settings = require("../../common/settings-access").settings;
+const logger = require("../../logwrapper");
+const userDatabase = require("../../database/userDatabase");
+const Mixplay = require("../../interactive/mixplay");
+
+// Active user toggle
+let activeUserListStatus = settings.getActiveMixplayUserList() === false ? false : true;
```

Comment:
This settings method already does a null/default value check so you dont' need to do it here. Also as it stands currently, `getActiveMixplayUserList()` actually returns a string ("On" or "Off") and not a boolean as you are checking here. See my comment where that method is defined, I suggest changing it to return a boolean to avoid confusion as has happened here.